core/primitives-core/src/profile.rs

```diff
@@ -157,146 +261,291 @@ impl fmt::Debug for ProfileData {
     }
 }
 
-#[derive(Clone, Copy, Debug)]
-pub enum Cost {
-    ActionCost { action_cost_kind: ActionCosts },
-    ExtCost { ext_cost_kind: ExtCosts },
-    WasmInstruction,
-}
+mod profile_v1 {
```

Comment:
Any reason to make this into a module? Maybe consider moving this into a separate file?

Reply:
It is now a separate file.